oap-stdlib/src/main/java/oap/http/Client.java

```diff
@@ -311,22 +311,9 @@ public Response post( String uri, byte[] content, String contentType, Map<String
 
     @SneakyThrows
     public Response uploadFile( String uri, RequestBody body, Map<String, Object> givenHeaders ) {
```

Comment:
givenHeaders -> is not met anywhere in similar methods - everywhere we have headers